packages/client/src/components/manage-users/table-cell-with-dialog.vue

```diff
@@ -1,6 +1,11 @@
 <template>
   <q-td>
-    <span v-if="value <= 0 && !clickableWhenZero" class="text-body2">
+    <span
+      v-if="disable || (value <= 0 && !clickableWhenZero)"
+      class="text-body2"
+      :class="{ 'cursor-not-allowed': disable }"
+      @click.stop
```

Comment:
Adding it here won't work, the `q-td` outside still gets the click and the dialog gets opened anyway

What you'd need to do is to conditionally add the click stop on the parent component

I did it in a project, but then I had to remove it and right now I can't remember where I did it, but the gist was to use dynamic event binding, something like:

`@[<condition> ? 'onClick.stop' : '']`

Reply:
Hmm, I see. `@click.stop` doesn't do anything when used on `q-td`, due to how attributes are merged. So, I defined the click event in emits and started emitting it conditionally. I also moved the cursor styles to q-td.